packages/client/tests/functional/interactive-transactions/tests.ts

```diff
@@ -113,6 +115,27 @@ testMatrix.setupTestSuite(({ provider }) => {
     expect(users.length).toBe(0)
   })
 
+  /**
+   * Transactions should fail and rollback if a value is thrown within
+   */
+  test('rollback throw value', async () => {
+    const result = prisma.$transaction(async (prisma) => {
+      await prisma.user.create({
+        data: {
+          email: 'user_1@website.com',
+        },
+      })
+
+      throw 'you better rollback now'
+    })
+
+    await expect(result).rejects.toMatchInlineSnapshot(`you better rollback now`)
```

Comment:
```suggestion
    await expect(result).rejects.toBe(`you better rollback now`)
```

So we could be sure we caught a thrown string and not, for example, `Error` object with the same message.